cypher-shell/src/main/java/org/neo4j/shell/cli/InteractiveShellRunner.java

```diff
@@ -154,7 +155,14 @@ public void handle(final Signal signal) {
             executer.reset();
         } else {
             // Print a literal newline here to get around us being in the middle of the prompt
-            logger.printError(AnsiFormattedText.s().colorRed().append("\nKeyboardInterrupt").formattedString());
+            logger.printError(
+                    AnsiFormattedText.s().colorRed()
+                            .append("\n" + "Interrupted (Note that Cypher queries must end with a ")
+                            .bold().append("semicolon. ").boldOff()
+                            .append("Type ")
+                            .bold().append(Exit.COMMAND_NAME).append(" ").boldOff()
```

Comment:
Do you feel strongly about this? I just felt like there's a bit of consistency when all the `append("Text")` start without spaces. I'd much rather have it this way instead of `append(" Text")`